assets_management/tests/test_assets_common.py

```diff
@@ -0,0 +1,310 @@
+        )
+        # Accounts
+        cls.asset_account_company1 = cls.env["account.account"].search(
+            [
+                (
+                    "user_type_id",
+                    "=",
+                    cls.env.ref("account.data_account_type_fixed_assets").id,
+                ),
+                ("company_id", "=", cls.company1.id),
+            ],
+            limit=1,
+        )
+        cls.asset_account_company2 = cls.asset_account_company1.copy(
+            {"company_id": cls.company2.id}
+        )
+
+        cls.depreciation_account_company1 = cls.env["account.account"].search(
+            [
+                (
+                    "user_type_id",
+                    "=",
+                    cls.env.ref("account.data_account_type_expenses").id,
+                ),
+                ("company_id", "=", cls.company1.id),
+            ],
+            limit=1,
+        )
+        cls.depreciation_account_company2 = cls.depreciation_account_company1.copy(
+            {"company_id": cls.company2.id}
+        )
+
+        cls.fund_account_company1 = cls.env["account.account"].search(
+            [
+                (
+                    "user_type_id",
+                    "=",
+                    cls.env.ref("account.data_account_type_non_current_assets").id,
+                ),
+                ("company_id", "=", cls.company1.id),
+            ],
+            limit=1,
+        )
+        cls.fund_account_company2 = cls.fund_account_company1.copy(
+            {"company_id": cls.company2.id}
+        )
+
+        cls.gain_account_company1 = cls.env["account.account"].search(
+            [
+                (
+                    "user_type_id",
+                    "=",
+                    cls.env.ref("account.data_account_type_revenue").id,
+                ),
+                ("company_id", "=", cls.company1.id),
+            ],
+            limit=1,
+        )
+        cls.gain_account_company2 = cls.gain_account_company1.copy(
+            {"company_id": cls.company2.id}
+        )
+
+        cls.loss_account_company1 = cls.env["account.account"].search(
+            [
+                (
+                    "user_type_id",
+                    "=",
+                    cls.env.ref("account.data_account_type_expenses").id,
+                ),
+                ("company_id", "=", cls.company1.id),
+            ],
+            limit=1,
+        )
+        cls.loss_account_company2 = cls.loss_account_company1.copy(
+            {"company_id": cls.company2.id}
+        )
+        # Journals
+        cls.journal_company1 = cls.env["account.journal"].search(
+            [("type", "=", "general"), ("company_id", "=", cls.company1.id)],
+            limit=1,
+        )
+        cls.journal_company2 = cls.journal_company1.copy(
+            {"company_id": cls.company2.id}
+        )
+        # Asset categories
+        cls.asset_category_1_company1 = cls.env["asset.category"].create(
```

Comment:
**nitpick:** Preferisco personalmente avere i nomi delle variabili più consistenti tra di loro, potresti mettere gli underscore anche per i numeri?

https://peps.python.org/pep-0008/#function-and-variable-names